inc/class-cachify.php

```diff
@@ -514,6 +546,30 @@ public static function robots_txt( $data ) {
 		return $data;
 	}
 
+	/**
+	 * HDD Cache expiration cron action
+	 *
+	 * @since  2.3.3
+	 */
+	public static function run_hdd_cache_cron() {
+		Cachify_HDD::clear_cache();
+	}
+
+	/**
+	 * Add cache expiration cron shedule
+	 *
+	 * @since  2.3.3
+	 *
+	 * @return array
+	 */
+	public static function add_cron_cache_expiration() {
```

Comment:
The method signature is missing the  `$schedules` parameter, so it always creates a new single-element array.

Reply:
Missed that, fixed it now